docs/.vuepress/config.ts

```diff
@@ -174,6 +174,58 @@ export default defineUserConfig<DefaultThemeOptions>({
             ],
           },
         ],
+      },
+	  "/zh_t/": {
+        selectLanguageName: "中文（繁体）",
+        navbar: [
+          {
+            text: "貢獻代碼",
+            link: "https://github.com/solana-developers/solana-cookbook",
+          },
+        ],
+        sidebar: [
+          {
+            text: "開始使用",
+            children: ["/zh_t/", "/zh_t/getting-started/installation.md"],
+          },
+          {
+            text: "核心概念",
+            children: [
+              "/zh_t/core-concepts/accounts.md",
+              "/zh_t/core-concepts/programs.md",
+              "/zh_t/core-concepts/transactions.md",
+              "/zh_t/core-concepts/pdas.md",
+            ],
+          },
+          {
+            text: "指南",
+            children: [
+              // '/zh_t/guides/get-program-accounts.md',
```

Comment:
Don't forget to uncomment once you've added each. Once this is fixed I'll merge

Reply:
Just fixed, thanks.